python/lsst/ts/hexrotcomm/config_schema.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of ts_hexrotcomm.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
```

Comment:
I think we agreed on `Vera Rubin Observatory` but I'll leave it up to you to decide if you want to update this and the other files.

Reply:
I hope "Developed for Vera C. Rubin Observatory Telescope and Site Systems." will do because it's what I used for all the other packages in this ticket. I looked at lsst.org and it seems to use "Rubin Observatory", which is a lot shorter.